README.ipynb

```diff
@@ -1015,7 +1015,7 @@
     "  --args=\"--threads=8,--timeout=0,main:app\" \\\n",
     "  --region=\"{region}\" \\\n",
     "  --memory=\"1G\" \\\n",
-    "  --no-allow-unauthenticated \\"
+    "  --no-allow-unauthenticated"
```

Comment:
I think it would be helpful to add a comment here after this `deploy to cloud run` step to say that once you see it deployed (can check through UI) then you can stop this cell (because it keeps running) so you can continue to run the next cells